tests/patmat/patmatexhaust.check

```diff
@@ -2,6 +2,7 @@
 11: Pattern Match Exhaustivity: Bar(_)
 23: Pattern Match Exhaustivity: (Qult(), Qult()), (Kult(_), Kult(_))
 49: Pattern Match Exhaustivity: _: Gp
+53: Pattern Match Exhaustivity: _
```

Comment:
This is the resulting of mirroring `scalac` behaviour more closely - guard on [this](https://github.com/AleksanderBG/dotty/blob/b66def8805200ae2e01e0a436b331613cb37880a/tests/patmat/patmatexhaust.scala#L55) line is constant-folded to `false`.